packages/p2p/src/components/my-ads/my-ads.jsx

```diff
@@ -61,27 +78,36 @@ class MyAds extends React.Component {
                 </div>
             );
         }
+
         return (
-            <div className='p2p-cashier__empty'>
+            <div
+                className={classnames('p2p-cashier__empty', 'p2p-my-ads__empty', {
+                    'p2p-my-ads__empty--pending': this.state.is_pending,
+                })}
+            >
                 <Icon icon='IcCashierSendEmail' className='p2p-cashier__empty-icon' size={102} />
                 <div className='p2p-cashier__empty-title'>
-                    <Localize i18n_default_text='Want to post ads?' />
+                    {this.state.is_pending ? localize('Documents received') : localize('Want to post ads?')}
                 </div>
-                <div className='p2p-cashier__empty-contact'>
-                    <Localize
-                        i18n_default_text='Email <0>{{support_email}}</0>'
-                        values={{ support_email: `p2p-support@${this.context.email_domain}` }}
-                        components={[
-                            <a
-                                key={0}
-                                className='link'
-                                rel='noopener noreferrer'
-                                target='_blank'
-                                href={`mailto:p2p-support@${this.context.email_domain}`}
-                            />,
-                        ]}
-                    />
+                <div className='p2p-cashier__empty-text'>
+                    {!this.state.is_pending && (
+                        <p>
+                            <Localize i18n_default_text='Register with us here.' />
+                        </p>
+                    )}
+                    <p>
+                        {this.state.is_pending
+                            ? localize('We will contact you once we have verified the information provided.')
+                            : localize('We’ll need you to upload your documents to verify your identity and address.')}
+                    </p>
                 </div>
+                <a href='/account/proof-of-identity' className='p2p-cashier__empty-button'>
+                    <Button
+                        type='button'
+                        text={this.state.is_pending ? localize('Check Status') : localize('Apply')}
```

Comment:
Should status be in sentence case?

```suggestion
                        text={this.state.is_pending ? localize('Check status') : localize('Apply')}
```